docs/classic-ui/images.md

```diff
@@ -237,16 +237,35 @@ image_scale = scaling_util.publishTraverse(context.REQUEST, groups[1])
 
 ## Scaling `direction`
 
-The default direction is `thumbnail`.
+Scaling is intended for the optimal display of images in a web browser.
 
-Other options are:
+To scale an image, you can use the `direction` parameter to control the scaling output.
+You must use either `width` or `height`, or both.
 
-* `down`
-* `keep`
-* `scale-crop-to-fill`
-* `scale-crop-to-fit`
-* `thumbnail`
-* `up`
+Three different scaling options are supported.
+They correspond to the CSS [`background-size`](https://developer.mozilla.org/en-US/docs/Web/CSS/background-size) values.
+
+The possible options for `direction` are listed below, where the default option is `scale`.
+
+`scale`
+:   This is the default option.
+    `scale` scales to the requested dimensions without cropping.
+    The resulting image may have a different size than requested.
+    This option requires both `width` and `height` to be specified.
+    It does not scale up.
+
+    Deprecated spellings: `keep`, `thumbnail`.
+
+`contain`
+:   `contain` starts by scaling the image either to the smaller dimension when you give both `width` and `height`, or to the only given dimension, then crops to the other dimension if needed.
```

Comment:
I rewrote this to clarify what I think it does, but I am not sure. Please double-check and correct me.